controls/radwizard/getting-started.md

```diff
@@ -10,7 +10,7 @@ position: 2
 
 # Getting Started
 
-This tutorial will walk you through the creation of a sample application that contains __RadWizard__ control.
+This tutorial will walk you through the creation of a sample application that contains the __RadWizard__ control.
 
 >In order to use the __RadWizard__ control in your projects you have to add references to the following assemblies:
```

Comment:
Here you can use a new section "Assembly References" as in RadGridView Getting Started article.